lib/models/entitlement_verification_mode.dart

```diff
@@ -0,0 +1,29 @@
+/// Enum of entitlement verification modes
+enum EntitlementVerificationMode {
+  /// The SDK will not perform any entitlement verification.
+  disabled,
+
+  /// Enable entitlement verification.
+  ///
+  /// If verification fails, this will be indicated with [VerificationResult.FAILED] in
+  /// the [EntitlementInfos.verification] and [EntitlementInfo.verification] properties but parsing will not fail
+  /// (i.e. Entitlements will still be granted).
+  ///
+  /// This can be useful if you want to handle verification failures to display an error/warning to the user
+  /// or to track this situation but still grant access.
+  informational,
+
+  // Disabled temporarily until ENFORCED is supported.
+  // enforced
+}
+
+extension EntitlementVerificationModeExtension on EntitlementVerificationMode {
```

Comment:
Note that this is something new. Basically, we're trying to pass an enum back to the native layer, which is not something we did before (afaik). Instead of using freezed to perform this mapping, I'm hardcoding it here, since it seemed easier.